### ISSUE TITLE
Buddy client extensions

### ISSUE THREAD

**Issue by bic-user** [lukeweber/webrtc-jingle-client#101]
I'm trying to use webrtc-jingle as google talk client. Not all buddies can receive calls. I want to distinguish them. The only place with such info (that I found) is node "caps:c" in raw xml that is handled by XmppRosterContactImpl from libjingle. There is "ext" over there and it contains "voice" if call is possible. Is there a simpler way to get that than modify third-party lib and drag extension through all native client part?

**Comment by lukeweber** [comment]
Nope, you'd need to drag it through. Probably the most straight forward would be to modify android/voice-client-native/jni/voiceclient_main.cc -> OnSignalPresenceChanged

Probably include a boolean, or int of caps, depending on how much you want to encode. In the end, it will be quite a few files to change, but that's the bridge method between java and libjingle.

**Comment by bic-user** [comment]
thanks! worked for me. the one should search for "voice-v1" in extensions. But it doesn't work for new hangouts. I'll start separate thread for that.
